AndroidSDK/src/com/leanplum/LeanplumEditorMode.java

```diff
@@ -26,6 +26,7 @@
  *
  * @author Ben Marten
  */
+@Deprecated
```

Comment:
Please add a comment to the javadoc like: `Deprecation: This method will become package protected in a future release, since t is not intended to be public API.`